packages/frontend/core/src/blocksuite/presets/ai/peek-view/chat-block-peek-view.ts

```diff
@@ -450,12 +457,18 @@ export class AIChatBlockPeekView extends LitElement {
 
   override firstUpdated() {
     // first time render, scroll ai-chat-messages-container to bottom
-    requestAnimationFrame(() => {
-      if (this._chatMessagesContainer) {
-        this._chatMessagesContainer.scrollTop =
-          this._chatMessagesContainer.scrollHeight;
-      }
-    });
+    requestAnimationFrame(this._scrollMessageContainerToBottom);
+  }
+
+  override updated(changedProperties: PropertyValues) {
+    super.updated(changedProperties);
+    if (
+      changedProperties.has('chatContext') &&
```

Comment:
I think "scroll to the bottom with incoming message" should only happen if the content is at the bottom of the scroll container. This is the behavior of claude (or perhaps chatgpt web as well?)